client/src/dependencies/DependencyManager.ts

```diff
@@ -39,7 +39,21 @@ export class DependencyManager {
 
     }
 
+    // Need this function as proxyquire doesn't work
+    public requireGrpc(): void {
+        require('grpc');
+    }
+
     public async hasNativeDependenciesInstalled(): Promise<boolean> {
+        const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
+
+        try {
+            this.requireGrpc();
```

Comment:
This should loop through all the native dependencies in case we add more in the future

Reply:
Made it a bit more generic now!